app/src/main/java/com/kickstarter/services/ApiClient.java

```diff
@@ -32,11 +32,13 @@
 import com.kickstarter.services.apiresponses.CategoriesEnvelope;
 import com.kickstarter.services.apiresponses.CommentsEnvelope;
 import com.kickstarter.services.apiresponses.DiscoverEnvelope;
+import com.kickstarter.services.apiresponses.MessageThreadsEnvelope;
 import com.kickstarter.services.apiresponses.StarEnvelope;
 
 import java.util.Arrays;
 import java.util.List;
 
+import retrofit2.Response;
```

Comment:
yeah, since [this line](https://github.com/kickstarter/android-oss/pull/73/files/9a79e5c0bc5ee52805653e10ffb7b611818fd4ea#diff-33e9703158d672ad42df5e7fa677e9a1R208) is the first time we're using an intermediate variable for some additional logic on the `Response`